Formula/bazelisk.rb

```diff
@@ -39,7 +39,7 @@ def install
     # This is an older than current version, so that we can test that bazelisk
     # will target an explicit version we specify. This version shouldn't need to
     # be bumped.
-    ENV["USE_BAZEL_VERSION"] = "0.28.0"
-    assert_match "Build label: 0.28.0", shell_output("#{bin}/bazelisk version")
+    ENV["USE_BAZEL_VERSION"] = "4.1.0"
+    assert_match "Build label: 4.1.0", shell_output("#{bin}/bazelisk version")
```

Comment:
```suggestion
    bazel_version = Hardware::CPU.arm? ? "4.1.0" : "4.0.0"
    ENV["USE_BAZEL_VERSION"] = bazel_version
    assert_match "Build label: #{bazel_version}", shell_output("#{bin}/bazelisk version")
```
Something like this should work.